core/src/components/picker-internal/test/basic/index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8">
+    <title>Picker - Basic</title>
+    <meta name="viewport"
+      content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
+    <link href="../../../../../css/ionic.bundle.css" rel="stylesheet">
+    <link href="../../../../../scripts/testing/styles.css" rel="stylesheet">
+    <script src="../../../../../scripts/testing/scripts.js"></script>
+    <script type="module" src="../../../../../dist/ionic/ionic.esm.js"></script>
+    <style>
+      .grid {
+        display: grid;
+        grid-template-columns: repeat(3, minmax(250px, 1fr));
+        grid-row-gap: 20px;
+        grid-column-gap: 20px;
+      }
+      h2 {
+        font-size: 12px;
+        font-weight: normal;
+
+        color: #6f7378;
+
+        margin-top: 10px;
+        margin-left: 5px;
+      }
+
+      @media screen and (max-width: 800px) {
+        .grid {
+          grid-template-columns: 1fr;
+          padding: 0;
+        }
+      }
+
+      ion-popover {
+        --width: 300px;
+      }
+
+      ion-modal {
+        --height: 250px;
+      }
+    </style>
+  </head>
+
+  <body>
+    <ion-app>
+      <ion-header>
+        <ion-toolbar>
+          <ion-title>Picker - Basic</ion-title>
+        </ion-toolbar>
+      </ion-header>
+      <ion-content class="ion-padding">
+        <div class="grid">
+          <div class="grid-item">
+            <h2>Inline</h2>
+            <ion-picker-internal>
+              <ion-picker-column-internal id="first"></ion-picker-column-internal>
+              <ion-picker-column-internal id="second"></ion-picker-column-internal>
+            </ion-picker-internal>
+          </div>
+          <div class="grid">
+            <div class="grid-item">
+              <h2>Numeric Input</h2>
+              <ion-picker-internal>
+                <ion-picker-column-internal numeric-input="true" id="numeric-first"></ion-picker-column-internal>
+                <ion-picker-column-internal id="numeric-second"></ion-picker-column-internal>
```

Comment:
Would be good to have both a picker with one numeric and one non-numeric, as well as a picker with two numerics.